admin_scripts/ubuntu/ubuntu_16.04/print-clients/printer-add.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -e
+
+
+# Vars
+# Replace space with underscore and remove æøå
+printer_name=`echo $1 | tr ' ' '_' | tr -dc '[:print:]'`
+printer_conn=$2
+printer_loc=$3
+# Remove æøå
+printer_descr=`echo $4 | tr -dc '[:print:]'`
+printer_driver=$5
+
+
+# Define conncetion type for the printer
+if [ `echo $printer_conn | grep 1` ]
+then
+    printer_conn="socket://"
+
+elif [ `echo $printer_conn | grep 2` ]
+then
+    printer_conn="usb://"
+else
+    echo "Forbindelsestypen findes ikke"
+    exit 1
+fi
```

Comment:
I wonder sometimes if OS2borgerPC needs a multiple-choice option for script parameters, so that we could display this as a drop-down menu with `Netværk` and `USB` instead of these mystery numbers...

Reply:
:)